src/Symfony/Component/Form/FormErrorIterator.php

```diff
@@ -265,6 +266,16 @@ public function seek($position)
         }
     }
 
+    public function getFormErrorByCode($code)
```

Comment:
@HeahDude I would prefer return array/empty array, what name give for this method?